nyc.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  reporter: ["text", "lcov"],
+  "check-coverage": true,
+  all: true,
+  // https://reflectoring.io/100-percent-test-coverage/
```

Comment:
this comment does not seem to belong here
if you are not using enforcing 100% coverage